docs/DeveloperGuide.adoc

```diff
@@ -64,22 +68,22 @@ image::LogicClassDiagram.png[]
 [discrete]
 ==== How the architecture components interact with each other
 
-The _Sequence Diagram_ below shows how the components interact with each other for the scenario where the user issues the command `delete 1`.
+The _Sequence Diagram_ below shows how the components generically interact with each other for the scenario where the user issues the command `some command`.
 
-.Component interactions for `delete 1` command
+.Component Interactions for `some command` Command
 image::ArchitectureSequenceDiagram.png[]
```

Comment:
the "some command" in execute is missing the closing ", would be good to add it in